src/OpenTelemetry.Resources.Container/ContainerDetector.cs

```diff
@@ -40,25 +46,29 @@ internal enum ParseMode
     /// <returns>Resource with key-value pairs of resource attributes.</returns>
     public Resource Detect()
     {
-        var cGroupBuild = this.BuildResource(Filepath, ParseMode.V1);
-        if (cGroupBuild == Resource.Empty)
+        var resource = this.BuildResource(Filepath, ParseMode.K8s);
```

Comment:
Call directly, ExtractK8sContainerId.
Passing unused file path can be misleading. You can fully remove ParseMode.K8s.

Reply:
Thank you for the suggestion. I did something, can you please tell me if it is better?

Reply:
LGTM, thanks.